pkg/commands/config.go

```diff
@@ -61,11 +61,13 @@ func getBaseImage(bo *options.BuildOptions) build.GetBase {
 		//
 		// Platforms can be comma-separated if we only want a subset of the base
 		// image.
-		multiplatform := bo.Platform == "all" || strings.Contains(bo.Platform, ",")
-		if bo.Platform != "" && !multiplatform {
+		allPlatforms := len(bo.Platforms) == 1 && bo.Platforms[0] == "all"
+		selectiveMultiplatform := len(bo.Platforms) > 1
+		multiplatform := allPlatforms || selectiveMultiplatform
+		if len(bo.Platforms) > 0 && !multiplatform {
```

Comment:
I like the new descriptive variable names!

I think `bo.Platforms` is always non-empty at this point, because we set it to some default if unset. Can this just be `if !multiplatform`? That makes the `bo.Platforms[0]` easier to understand, since we'll know there's only one value.